### PR TITLE
Remove old Microsoft.Logic schema template

### PR DESCRIPTION
**Pull request by swylezol**
We already have a schema template for our GA version (2016-06-01), so we can delete the old one

### CODE REVIEW

File: schemas/2015-02-01-preview/Microsoft.Logic.json [hunk 1]
```diff
@@ -47,102 +37,9 @@
         "properties"
       ],
       "description": "Microsoft.Logic/workflows"
-    },
-    "workflows_accessKeys": {
```
Comment (TianoMS) [inline schemas/2015-02-01-preview/Microsoft.Logic.json]: please remove the references schemas/2014-04-01-preview/deploymentTemplate.json and schemas/2015-01-01/deploymentTemplate.json.

**Comment by TianoMS** [comment]
@swylezol One comment. Please address it.

**Comment by swylezol** [comment]
Thanks for pointing out @TianoMS. Removed the references.

File: schemas/2014-04-01-preview/deploymentTemplate.json [hunk 2]
```diff
@@ -51,13 +51,13 @@
...
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/labs_artifactsources" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/labs_customimages" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/labs_formulas" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/labs_policysets_policies" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/labs_schedules" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/labs_virtualmachines" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/labs_virtualnetworks" },
```
Comment (TianoMS) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: What's changed here?

Comment (swylezol) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: Nothing as far as I can see. Codeflow does not show any changes for those lines. Maybe line feed? Edited the files in VS Code.

Comment (swylezol) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: Yeah these lines had CR+LF, so it removed CR

**Pull request merged**